styles.css

```diff
@@ -0,0 +1,470 @@
+@import url("https://fonts.googleapis.com/css2?family=Exo+2:ital,wght@0,100..900;1,100..900&display=swap");
+
+/* Global styles */
+* {
+  box-sizing: border-box;
+  margin: 0;
+  padding: 0;
+  line-height: inherit;
+}
+body {
+  font-family: "Exo 2", sans-serif;
+  font-optical-sizing: auto;
+  font-weight: 400;
+  font-style: normal;
+  margin: 0;
+  background-color: #1d1d1d;
+  color: #ccc;
+  background-image: radial-gradient(#282828 0.1rem, transparent 0.15rem);
+  background-size: 4rem 4rem;
+}
+em {
+  font-style: italic;
+}
+h1,
+h2,
+h3 {
+  font-weight: 700;
+  color: hsl(210, 70%, 60%);
+  padding: 0.5rem 0;
+}
+ul {
+  padding: 0.5rem 0 0.5rem 1.5rem;
+}
+.header_bg {
+  position: relative;
+  width: 100%;
+  background-color: #69264a;
+  background-image: url("./assets/blender_viewport.webp");
+  background-position: center;
+}
+@media (min-width: 1920px) {
+  .header_bg {
+    background-size: cover;
+  }
+}
+.header_bg::before {
+  position: absolute;
+  top: 0;
+  left: 0;
+  height: 100%;
+  width: 100%;
+  content: "";
+}
+h1 {
+  margin: 0;
+  padding: 0;
+  padding-bottom: 25%;
+  background-image: url("./assets/slc_blender_user_group_logo.png");
+  background-size: contain;
+  position: relative;
+}
+h1 span {
+  position: absolute;
+  visibility: hidden;
+}
+.container {
+  margin: auto;
+  max-width: 960px;
+}
+.content {
+  padding: 0 0.8rem;
+}
+a,
+a:link,
+a:visited {
+  color: #e87d0d;
+}
+img {
+  max-width: 100%;
+  display: block;
+}
+
+input[type="radio"],
+input[type="checkbox"] {
+  display: none;
+}
```

Comment:
Can we prefix this effect with some kind of class or parent, like to only do this in certain parent chains? I think I actually prefer the idea of selecting these by their parent, rather than adding a class attribute on every input tag because that's smaller HTML.